dbmigrator/__init__.py

```diff
@@ -12,4 +13,12 @@
 logger.addHandler(handler)
 
 
-__all__ = ('logger',)
+def deferrable(func):
+    """A decorator to mark the migration as deferred by default.  The state of
+    the migration can be changed later by using `mark`.
+    """
```

Comment:
When I started writing it, it was meant to be given the default state of deferred, but then with the table we have right now it's not possible to "undefer" it... I need to add migrations for db-migrator. :p